docs/BreakingChanges/105 ETW EventListeners do not capture events from providers with explicit keywords.md

```diff
@@ -0,0 +1,27 @@
+## 105: ETW EventListeners do not capture events from providers with explicit keywords (like the TPL provider)
```

Comment:
{
  "Id": "105",
  "Title": "ETW EventListeners do not capture events from providers with explicit keywords (like the TPL provider)",
  "Details": "ETW EventListeners with a blank keyword mask do not properly capture events from providers with explicit keywords. In the .NET Framework 4.5, the TPL provider began providing explicit keywords and triggered this issue. In the .NET Framework 4.6, EventListeners have been updated to no longer have this issue.",
  "Suggestion": "To work around this problem, replace calls to EnableEvents(eventSource, level) with calls to the EnableEvents overload that explicitly specifies the \"any keywords\" mask to use: `EnableEvents(eventSource, level, unchecked((EventKeywords)0xFFFFffffFFFFffff))`.\n\nAlternatively, this issue has been fixed in the .NET Framework 4.6 and may be addressed by upgrading to that version of the .NET Framework.",
  "Link": "http://connect.microsoft.com/VisualStudio/feedback/details/816989/tpl-etw-events-for-task-not-captured-by-eventlistener-anymore",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Diagnostics.Tracing.EventListener.EnableEvents(System.Diagnostics.Tracing.EventSource,System.Diagnostics.Tracing.EventLevel)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.6",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": true,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Diagnostics.Tracing.EventListener.EnableEvents(System.Diagnostics.Tracing.EventSource,System.Diagnostics.Tracing.EventLevel)